### DISCUSSION TITLE
How to validate the mj-all attribute?

### DISCUSSION THREAD

**Discussion by kishanindegene** [mjmlio/mjml#2526] [Q&A] [answered]
`<mjml>
  <mj-head>
    <mj-attributes>
      <mj-all colcccccor="#000000" font-family="Arial"></mj-all>
    </mj-attributes>
</mj-head>
</mjml>`

**Comment by iRyusa** [comment, marked as answer]
For now there's no way to do so, it would require to merge every type attributes into an object and deal with different types.
